src/foam/nanos/controller/ApplicationController.js

```diff
@@ -137,6 +138,33 @@ foam.CLASS({
   `,
 
   properties: [
+    {
+      class: 'Enum',
+      of: 'foam.u2.layout.DisplayWidth',
+      name: 'displayWidth',
```

Comment:
Let's make this a property and/or function on the Window object itself.  foam.core.Window, or a refinement added in foam.u2.Element.

Reply:
Fixed all the other stuff, just needs the window refactor like you said.